sdkutil/broadcast.go

```diff
@@ -86,6 +87,34 @@ func BroadcastTX(ctx client.Context, flags *pflag.FlagSet, msgs ...sdk.Msg) erro
 
 }
 
+// PrepareFactory has been copied from cosmos-sdk to make it public.
```

Comment:
This is kind of weird? Why is it needed to copy a function from cosmos SDK into our code?

Reply:
That function was previously public, but `cosmos v0.43.0` made it private:
https://github.com/cosmos/cosmos-sdk/blame/release/v0.43.x/CHANGELOG.md#L114

This is being used by `BroadcastTX()` which is also a function originally defined in `cosmos-sdk` but slightly modified for a use-case of ours. So, there is no other option than to copy even the `PrepareFactory` function to our code.